config/solaris2.x

```diff
@@ -141,7 +141,7 @@ echo "C compiler '$CC' is $cc_version_info"
 case $F77 in
     # The PGI and Intel compilers are automatically detected below
     *f90*|*f77*)
-        fc_version_info=`$F77 $FFLAGS $H5_FFLAGS -V 2>&1 | grep 'Sun' |\
+        fc_version_info=`$F77 $FFLAGS $H4_FFLAGS -V 2>&1 | grep 'Sun' |\
```

Comment:
Not critical here, but I think we can rename this file to just 'solaris' like we do in HDF5